dash/dash.py

```diff
@@ -1208,6 +1208,8 @@ def dispatch(self):
             cb = self.callback_map[output]
             func = cb["callback"]
 
+            g.ignore_register_page = cb.get("long", False)
```

Comment:
Would it be possible to still catch `register_page` calls inside the callback functions, possibly by turning `ignore_register_page` back off again immediately before invoking the function itself, ie [here](https://github.com/plotly/dash/blob/648130685a9d4fee32e12459845997792d76b6f1/dash/long_callback/managers/celery_manager.py#L152-L157) and [here](https://github.com/plotly/dash/blob/648130685a9d4fee32e12459845997792d76b6f1/dash/long_callback/managers/diskcache_manager.py#L160-L165)?

Reply:
Good call :ok_hand: